admin/app/main/s3_client.py

```diff
@@ -39,9 +39,11 @@ def get_object(self, filename):
     def delete_object(self, filename):
         return self.get_object(filename).delete()
 
-    def copy_object(self, old_name, new_name):
+    def copy_object(self, old_name, new_name, acl=None):
```

Comment:
depends how explicit you want this to be. Personally I don't mind forwarding arguments to copy_from. you would do this with something like

```
def copy_object(self, old_name, new_name, **kwargs):
    ...copy_from(..., **kwargs)
```